src/find/matchers/ls.rs

```diff
@@ -256,7 +257,7 @@ impl Ls {
 impl Matcher for Ls {
     fn matches(&self, file_info: &WalkEntry, matcher_io: &mut MatcherIO) -> bool {
         if let Some(file) = &self.output_file {
-            self.print(file_info, file, true);
+            self.print(file_info, file.clone(), true);
```

Comment:
```suggestion
            self.print(file_info, file.as_ref(), true);
```

Same everywhere else, there's usually no need to clone the `Arc`